src/helpers/renderSvg.tsx

```diff
@@ -0,0 +1,3 @@
+export const renderSvg = ({ Component, ...props }: any) => (
```

Comment:
should the type here be something like:
```
{
  Component: React.ReactNode,
  [key: string]: any,
}
```
instead of `any`?

Reply:
TypeScript doesn't like `React.ReactNode` or `React.ReactElement` there. 🤔 It's on my list to make this function better or find another solution.

![image](https://user-images.githubusercontent.com/7346473/125851020-7ab6cf89-49ce-44ad-96b6-3fa5bb75d3c0.png)

Reply:
Ah, that error actually makes sense. This may be helpful: https://stackoverflow.com/questions/31815633/what-does-the-error-jsx-element-type-does-not-have-any-construct-or-call

Reply:
👍 Ideally, I'd like to make it SVG specific, if React and TypeScript will play along. 😊